R/mod-review-section.R

```diff
@@ -243,16 +245,24 @@ get_submission_list <- function(data) {
   sub_names <- purrr::map(
     sub_ids,
     function(x) {
-      paste0(data$submitted_on[data$form_data_id == x][1], 
-             "_", 
-             data$submission[data$form_data_id == x][1]
-      )
+      # Date string
+      date_string <- data$submitted_on[data$form_data_id == x][1]
+
+      # Step 1: Parse the string to a datetime object and convert to Eastern Time
```

Comment:
Do we want to indicate in the UI that the time zone is eastern time?

Reply:
Since we are using these dates as prefixes for dropdown menu items and table identifiers, we don't really have room to surface the time zone. Also, we are just using the date without the time, so I think it will be ok.